### PR TITLE
Updated dependencies for Debian buster

### PR DESCRIPTION
**Pull request by AhrazA**
The role was failing when installing the packages, as they no longer exist:
-   libgnome2-dev
-   libgnomeui-dev

On Debian Buster (10).

### CODE REVIEW

**Comment by AhrazA** [comment]
Maybe worth updating to 8.2 as well. I can include that in this PR if you want @mattladany

**Comment by mattladany** [comment]
> Maybe worth updating to 8.2 as well. I can include that in this PR if you want @mattladany

If you could update to the latest 8.2 as well that'd be great! I'll merge right after.

**Comment by AhrazA** [comment]
Done :)

**Review by mattladany — APPROVED** [review]

**Comment by mattladany** [comment]
> 
> 
> Done :)

Thank you!!

**Pull request merged**
